en/docs/debug.md

```diff
@@ -164,58 +146,39 @@ Each configuration supports a different set of attributes, which will be provide
 
 The following attributes are mandatory for all configurations.
 
-- `name` - The reader-friendly name to appear in the Debug launch configuration drop-down menu.
-- `type` - The type of debugger to use for this launch configuration. The attribute value must be kept as `ballerina` for all Ballerina debugging configuration types.
-- `request` - The request type of this launch configuration. Currently, `launch` and `attach` are supported.
+| Attribute                     	| Description                                                          	|
+|---------------------------------	|----------------------------------------------------------------------	|
+| `name`                 	| The reader-friendly name to appear in the Debug launch configuration drop-down menu                                  	|
+| `type`  	| The type of debugger to use for this launch configuration. The attribute value must be kept as `ballerina` for all Ballerina debugging configuration types                   	|
```

Comment:
for example here - the first sentence in the description has a full stop and the second one doesn't.

Reply:
Added the full stops back.